src/cocoa/toga_cocoa/window.py

```diff
@@ -221,15 +215,51 @@ def set_content(self, widget):
         )
         widget.native.addConstraint(self._min_height_constraint)
 
+    def get_title(self):
+        return str(self.native.title)
+
     def set_title(self, title):
         self.native.title = title
 
+    def get_position(self):
+        # If there is no active screen, we can't get a position
+        if len(NSScreen.screens) == 0:
+            return (0, 0)
+
+        primary_screen = NSScreen.screens[0].frame
+        window_frame = self.native.frame
+
+        # macOS origin is bottom left of screen, and the screen might be
+        # offset relative to other screens. Adjust for this.
+        return (
+            primary_screen.origin.x + window_frame.origin.x,
+            primary_screen.origin.y + primary_screen.size.height - (
+                window_frame.origin.y + window_frame.size.height
+            )
```

Comment:
Since screens[0] is guaranteed to be the primary screen, its origin will always be (0,0). From Apple's docs on `NSScreen.screens`:

> The screen at index 0 in the returned array corresponds to the primary screen of the user’s system. This is the screen that contains the menu bar and whose origin is at the point (0, 0).

The addition could therefore be simplified by dropping the origin.

Reply:
Good catch. Fix in #1462; And yes, I'd very much like to get code reviews into the process; that's difficult to do at present when I can't guarantee availability of anyone else. This will hopefully change soon.